js/cell.js

```diff
@@ -64,6 +68,10 @@ Cell.prototype.setPosition = function( x ) {
   this.renderPosition( x );
 };
 
+Cell.prototype.setAriaHidden = function ( val ) {
+  this.element.setAttribute('aria-hidden', val);
+};
```

Comment:
Oh ok, I tried to adhere to the stuff inside the parens but totally missed the spacing between the functions, thanks for pointing that out!